### ISSUE TITLE
Some annotations in instances_trainrem.json have no iscrowd key - are they all instances?

### ISSUE THREAD

**Issue by JohannesTheo** [kdexd/coco-rem#5]
Hey there, first of all, thank you for the great dataset! I'm actively using it in my work :)

Quick questions, just to be sure (probably no impact in practice). Is any annotation without `'iscrowd'` key a normal instance, i.e. not a crowd?

Since a part of my code relied on the presence of `'iscrowd'` in the annotations, I found that `354674/1093027` annotations in `instances_trainrem.json` have no `'iscrowd'` set (`7148/1093027` are crowd annotations btw). Typically, e.g. in mmdet, any annotation without `'iscrowd'` will be treated as instance, i.e. as not a crowd, similar to if `'iscrowd: 0'` is set. Is this assumption correct?

Thanks, Johannes

**Comment by ydvaayan** [comment]
Hey Johannes, thanks for using our annotations.
You can treat any instance without `iscrowd` field as instance annotations i.e. `iscrowd:0` . In train set 354674 instances are imported from LVIS dataset. The are instance annotations. Missing `iscrowd:0` is probably an oversight on our part. Thanks for pointing that out !

**Comment by JohannesTheo** [comment]
Hey @ydvaayan, thx for the fast response! :)

I see, that makes sense. As already mentioned, I don't think it has any practical implications as all detection frameworks I'm aware of will handle this case gracefully during training. In `instances_valrem.json` all fields are set properly. I just came across this becaus I track some metrics on train data as well. This is a custom thing however which no framework does by default and if anyone runs into it as I did, it's a trivial fix.
